source/apps.html.md.erb

```diff
@@ -35,8 +35,8 @@ Apps can be loosely grouped into different types, including:
 
 | Hosting | Count | Applications |
 | --- | --- | --- |
-<% Hosts::hosters_descending(Repos.active_apps).each do |id, label:, apps:| %>
```

Comment:
Re your commit message, this is failing because Ruby 2.7 would expand a hash into the keyword arguments, whereas Ruby 3.1 won't and without something to convert the hash to kwargs.

Your change looks like an improvement in my opinion.

Reply:
Oh, thank you! That's makes sense.